src/opm/input/eclipse/EclipseState/Grid/NNC.cpp

```diff
@@ -237,5 +354,11 @@ bool is_neighbor(const EclipseGrid& grid, std::size_t g1, std::size_t g2) {
             return {};
     }
 
+    KeywordLocation NNC::editr_location([[maybe_unused]] const NNCdata& nnc) const {
```

Comment:
up for debate i know but this is not maybe_unused this is unused and should be anonymized imo (possibly with variable name as a comment).

Reply:
changed her and for the other location functions.